about.html

```diff
@@ -101,12 +73,12 @@ <h1 class="past-seminar-header">See past SCDK GIS Seminars</h1>
              which took place in Ibadan.
         </p>
         <div class="past-seminars-images">
-            <img src="images/conference1.JPG" alt="past seminar">
-            <img src="images/conference2.JPG" alt="past seminar">
+          <img src="images/conference1.png" alt="past seminar">
+          <img src="images/conference2.png" alt="past seminar">
```

Comment:
- [ ] Hey, you're closed on the required changes for this part, Please, kindly compare your color with the original to make it close below:

:point_down:  **YOUR DESIGN**

![yourOwn](https://user-images.githubusercontent.com/59397159/165743880-1c1881db-de60-4a08-8276-477441edfa7a.png)

:point_down:  **ORIGINAL COLOR DESIGN**
![red](https://user-images.githubusercontent.com/59397159/165743874-f88a9899-c6a6-4fd5-9373-cb61b144584d.png)